test/mock-connector.py

```diff
@@ -38,6 +39,27 @@
     level=logging.INFO,
     datefmt='%Y-%m-%d %H:%M:%S')
 
+def __parse_update_message(msg_file):
```

Comment:
Remove this function

Reply:
Because of the facts already explained, I can not remove.